azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+# https://aka.ms/yaml
```

Comment:
Could you please move this file into _.github/workflows/_?

Reply:
It don't belong to github action.I think better put in root directory as other open source project.

Reply:
And I don't sure will this make error for github action.